SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -49,6 +49,11 @@
   private final String logrotateExtrasDateformat;
   private final String[] logrotateExtrasFiles;
 
+  /**
+   * Extra files to backup to S3 besides the service log.
+   */
+  private final String[] s3FilesToBackup;
```

Comment:
should probably be explicit and call it `additionalS3FilesToBackup`, since we're doing the service.log file by default